etc/sh/interactive.d/80-aliases.sh

```diff
@@ -20,6 +20,7 @@ alias watch='watch '
 
 # Random other aliases:
 alias cc='gcc -std=c11 -pedantic -Wall -Wextra -Wbad-function-cast -Wcast-align -Wcast-qual -Wconversion -Wfloat-equal -Wformat=2 -Wlogical-op -Wmissing-declarations -Wmissing-prototypes -Wpointer-arith -Wshadow -Wstrict-prototypes -Wwrite-strings'
+alias fixinput='sh $XDG_CONFIG_HOME/karuiwm/profile.d/10-xinput.sh && sh $XDG_CONFIG_HOME/X11/xprofile.d/10-setxkbmap.sh'
```

Comment:
While marked as "tmp", I would actually like to keep this around here as well. The usecase is common enough that I expect this to happen also in my personal environment.

Sure something more elegant (see #34) might be a better longterm solution, but… welp, longterm solutions also usually take a long time to grow.

Reply:
Removed "tmp" marker (and also reworded commit message to reflect that it's an alias, not a script).

Reply:
LGTM